src/mcs/mcs_types.jl

```diff
@@ -31,7 +31,14 @@ struct TransformSpec
     end 
 end
 
-const CorrelationSpec = Tuple{Symbol, Symbol, Float64}
+"""
+Defines a target rank correlation to establish between the two named random vars.
+"""
+struct CorrelationSpec
+    name1::Symbol
+    name2::Symbol
+    value::Float64
+end
```

Comment:
I assume this is all done, but in general we could add support for this later: we don't have any model at the moment that needs this.

Reply:
This is indeed done; I ported my Python code, which already included this. I just changed a `const` declaration of a 3-tuple to a proper struct.